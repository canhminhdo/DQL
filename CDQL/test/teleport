#!/bin/sh

DIR=test
cd ../ && maude < teleport.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/teleport.out
diff $DIR/teleport.expected $DIR/teleport.out
rm $DIR/teleport.out