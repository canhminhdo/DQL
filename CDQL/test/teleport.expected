==========================================
reduce in TELEPORT : initQState |= [H(1) ; CX(1,2) ; encap(Alice || Bob)[c02 <:
    0, c02 <: 1, c12 <: 0, c12 <: 1, c02 :> 0, c02 :> 1, c12 :> 0, c12 :> 1]]P(
    2,qubitAt(initQState, 0)) .
result JudgmentSet: emptyJS
Bye.
